src/components/header/Header.js

```diff
@@ -54,30 +59,44 @@ function Header() {
       <HeaderWrapper wrapMobileMenu={mobileWidth} isMenuOpen={isMenuOpen}>
         <ul className="header__navbar">
           <li className="header__list">
-            <Link className="header__link" to="/dummy-page" smooth={true} onClick={onNavClick}>
-              We can help!
-            </Link>
-          </li>
-          <li className="header__list">
-            <Link className="header__link" to="/dummy-page" smooth={true} onClick={onNavClick}>
-              About
-            </Link>
-          </li>
-          <li className="header__list">
-            <Link className="header__link" to="/dummy-page" smooth={true} onClick={onNavClick}>
-              How to start
-            </Link>
-          </li>
-          <li className="header__list">
-            <Link className="header__link" to="/dummy-page" smooth={true} onClick={onNavClick}>
-              Projects
+            <Link
```

Comment:
This set up of links makes sense. The logic seems pretty straighforward. I'm getting some error when I manually update the URL, sometimes I'm not getting the updated link text back. If I click on card, the header text changes, but then when I edit the URL to go back to the homepage, the text doesn't update. This might be fixed when the routing changes, so no worries right now, but I thought I'd mention that first. 

An alternative method of creating these links is to create a new Array of header links to map over. There'd be an array of 2 objects, one with 5 links and the corresponding text, one with two links and the corresponding texts. One App level route would feed the data from one object in into the Header component, and the other route would feed the data from the other object. That way, when one route is shown, a header always has the right text, and when another route is shown, the header always has the right text.